RecoBTag/DeepFlavour/python/DeepFlavourTagInfos_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
filename pfDeepFlavourTagInfos_cfi.py
.. better yet get rid of this altogether to be delivered instead by the fillDescriptions

Reply:
@slava77, do you mean using the fillDescritions to put the default values? could you point me to a canonical example of this in CMSSW?

Reply:
for the "... better yet" part yes.
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp
is the primary reference

Reply:
using fillDescriptions since https://github.com/cms-sw/cmssw/pull/19893/commits/c2dc5c01427a4ea10ed8cba83f44382c986ebb46